src/Identity/test/Identity.Test/IdentityUIScriptsTest.cs

```diff
@@ -17,6 +17,7 @@
 
 namespace Microsoft.AspNetCore.Identity.Test;
 
+[SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/38542", Queues="OSX.1015.Amd64.Open;OSX.1015.Amd64")] //slow
```

Comment:
☹️

Note I'm skipping the test class in case the failure would just move to `IdentityUI_ScriptTags_SubresourceIntegrityCheck(...)` if I skipped only `IdentityUI_ScriptTags_FallbackSourceContent_Matches_CDNContent(...)`. I double-checked and don't see evidence `IdentityUI_ScriptTags_SubresourceIntegrityCheck(...)` ran before (and of course not after) the timeouts of `IdentityUI_ScriptTags_FallbackSourceContent_Matches_CDNContent(...)`.

Reply:
Yeah seems fine, we could go so far as skip All.OSX if you wanted since we really only need to run this on any single queue to ensure we are including the right file versions

Reply:
`<whining>I don't wanna do another iteration</whining>` 😭

Plus, we haven't seen similar issues in other queues. We could add a `[RunOnlyOnLatestUbuntu]` attribute at some point in the future 😄